docs/first-order/receptive_fields.ipynb

```diff
@@ -23,7 +23,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    dandi_api_key = ""
</pre><p>same as in optotagging on other PR. Doesn't this cause an error?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/237/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>